RELEASE-NOTES.md

```diff
@@ -2,9 +2,9 @@ Release 3.2
 ===========
 In release 3.2, the goal is to produce an update to the code given the large number of improvements since the previous release.
 
-There are a few bugs that will survive this release, most notably in the AVLTreeDigest. These have to do with large nubmers of repeated data points and are not new bugs.
+There are a few bugs that will survive this release, most notably in the AVLTreeDigest. These have to do with large numbers of repeated data points and are not new bugs.
  
-There is a also lot of work going on with serialization. I need to hear from people about  what they are doing with serialization so that we can build some test cases to allow an appropriate migration strategy to future serialization.
+There is an also lot of work going on with serialization. I need to hear from people about  what they are doing with serialization so that we can build some test cases to allow an appropriate migration strategy to future serialization.
```

Comment:
This fix isn't right. Should be "There is also a lot ..."

Reply:
Thanks for spotting this @tdunning . Edited the commit :+1: thanks!